roles/elastic-stack/ansible-elasticsearch/templates/instances.yml.j2

```diff
@@ -6,10 +6,10 @@
 instances:
 {% for (key,value) in instances.items() %}
 - name: "{{ value.name }}"
-{% if value.ip %}
+{% if value.ip is defined %}
```

Comment:
Hi @xr09 , this conditional doesn't check if the variable is empty.

After reviewing this, I also noticed that the `instances` variable is only defined in the playbook and is not initialized in the `/defaults` folder.

`Please add this variable to the `/defaults` folder and initialize them to `""`, this way we only have to check if the value is not empty with `value.ip | length > 0` for example

Thanks!

Reply:
@jm404  Just pushed new changes, could you take a look?